src/ui_components/search_input.js

```diff
@@ -102,6 +103,9 @@ export default class SearchInput {
     } else if (selectedItem instanceof Category) {
       window.app.navigateTo(`/places/?type=${selectedItem.name}`,
         {}, { replace: replaceUrl });
+    } else if (selectedItem instanceof Intention) {
```

Comment:
As the triggered action is always the same (calling `window.app.navigateTo`), I think we should just set two vars in each if statement : `url` and `options`, and call `window.app.navigateTo(url, options, { replace: replaceUrl });` at the end